.github/workflows/x-keycloak-quickstarts.yml

```
@@ -0,0 +1,33 @@
+name: X Keycloak QuickStarts
+
+on:
+  workflow_call:
+    inputs:
+      tag:
+        required: true
+        type: string
+      target-branch:
+        required: true
+        type: string
+    secrets:
+      GH_ORG:
+        required: true
+      GH_TOKEN:
+        required: true
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3.0.0
```

This will not allow minor and patch releases with bugfixes, perhaps we can use a semver major range instead?

```suggestion
        uses: actions/checkout@v3
```